projects/plugins/protect/src/js/components/firewall-header/index.jsx

```diff
@@ -0,0 +1,138 @@
+		setShowPopover( true );
+	}, [] );
+
+	const handleOut = useCallback( () => {
+		setShowPopover( false );
+	}, [] );
+
+	if ( 'on' === status ) {
+		return (
+			<AdminSectionHero>
+				<Container
+					className={ styles[ 'firewall-header' ] }
+					horizontalSpacing={ 6 }
+					horizontalGap={ 0 }
+				>
+					<Col>
+						<Text className={ classnames( styles.status, styles.active ) } variant={ 'label' }>
+							{ __( 'Active', 'jetpack-protect' ) }
+						</Text>
+						<H3 className={ styles[ 'firewall-heading' ] } mb={ 2 } mt={ 2 }>
+							{ __( 'Automatic firewall is on', 'jetpack-protect' ) }
+						</H3>
+					</Col>
+					<Col>
+						<div className={ styles[ 'stat-card-wrapper' ] }></div>
+					</Col>
+				</Container>
+			</AdminSectionHero>
+		);
+	}
+
+	if ( 'off' === status ) {
+		return (
+			<AdminSectionHero>
+				<Container
+					className={ styles[ 'firewall-header' ] }
+					horizontalSpacing={ 6 }
+					horizontalGap={ 0 }
+				>
+					<Col>
+						<Text className={ styles.status } variant={ 'label' }>
+							{ __( 'Inactive', 'jetpack-protect' ) }
+						</Text>
+						<H3 className={ styles[ 'firewall-heading' ] } mb={ 2 } mt={ 2 }>
+							{ __( 'Automatic firewall is off', 'jetpack-protect' ) }
+						</H3>
+						{ ! hasRequiredPlan && (
+							<>
+								<div className={ styles[ 'manual-rules-notice' ] }>
+									<Text variant={ 'body-small' } weight={ 600 }>
+										{ __( 'Only manual rules will be applied', 'jetpack-protect' ) }
+									</Text>
+									<div
+										className={ styles[ 'icon-popover' ] }
+										onMouseLeave={ handleOut }
+										onMouseEnter={ handleEnter }
+										onClick={ handleEnter }
+										onFocus={ handleEnter }
+										onBlur={ handleOut }
+										role="presentation"
+									>
+										<Icon icon={ help } />
+										{ showPopover && (
+											<Popover noArrow={ false } offset={ 5 }>
+												<Text className={ styles[ 'popover-text' ] } variant={ 'body-small' }>
+													{ __(
+														'The free version of the firewall only allows for use of manual rules.',
+														'jetpack-protect'
+													) }
+												</Text>
+											</Popover>
+										) }
+									</div>
+								</div>
+								<Button onClick={ getScan }>
+									{ __( 'Upgrade to enable automatic rules', 'jetpack-protect' ) }
+								</Button>
+							</>
+						) }
+					</Col>
+					<Col>
+						<div className={ styles[ 'stat-card-wrapper' ] }></div>
+					</Col>
+				</Container>
+			</AdminSectionHero>
+		);
+	}
+
+	return (
+		<AdminSectionHero>
+			<Container
+				className={ styles[ 'firewall-header' ] }
+				horizontalSpacing={ 6 }
+				horizontalGap={ 0 }
+			>
+				<Col>
+					<Spinner className={ styles.spinner } />
+					<H3 className={ styles[ 'firewall-heading' ] } mb={ 2 } mt={ 2 }>
+						{ __( 'Automatic firewall is being set up', 'jetpack-protect' ) }
+					</H3>
+					<Text variant={ 'body-small' } weight={ 600 }>
```

Comment:
This text size should be `18px`, I'm not sure if there's a `variant` for that. We might need some custom styles for this.